__TESTS__/unit/actions/Overlay.test.ts

```diff
@@ -0,0 +1,33 @@
+import {fill} from "../../../src/actions/resize/Resize";
+import {addImageLayer} from "../../../src/actions/layers/overlay";
+import {imageSource} from "../../../src/actions/layers/Sources";
+import TransformableImage from "../../../src/transformation/TransformableImage";
+
+describe('Tests for overlay actions', () => {
+  it('Parses an overlay with an imageSource', () => {
+    const tImage = new TransformableImage('sample');
+    tImage
+      .setConfig({
+        cloud: {
+          cloudName: 'demo'
+        }
+      })
+      .resize(fill(1000, 1000))
+      .overlay(
+        addImageLayer(
```

Comment:
I suggest to keep the signature of (source/layer, position/gravity, blendMode)

Reply:
In JS we need an extra layer: `addImageLayer` which is the actual action.

`addImageLayer` will accept (probably) three arguments:

Source (`imageSource`), `Position` and `blendMode`

this PR doesn't contrast that, but just implements the first part of the overlay